src/Components/Navbar.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Axios from 'axios';
+
+
+class Navbar extends React.Component {
+   constructor(props) {
+      super(props);
+      this.logout = this.logout.bind(this);
+   }
+
+
+
+   logout() {
+      // console.log('logout pressed');
+      // console.log(this.props.dataStore);
+      const config = {
+         headers: { 'Authorization': "Bearer " + this.props.token }
+      };
+      Axios.get('http://127.0.0.1:8000/api/logout', config)
+         .then(res => {
+
+            localStorage.removeItem('token');
+            localStorage.removeItem('data');
+            // this.props.tokenFunction("");
```

Comment:
Can get rid of this, seems like you switched over to the new datastore function (which is good!).